components/instantiation/NetPyNEInstantiated.js

```diff
@@ -78,18 +78,26 @@ export default class NetPyNEInstantiated extends React.Component {
         });
     }
 
-    plotFigure(pythonFigureMethod, plotName) {
-        Utils.sendPythonMessage(pythonFigureMethod, [])
+    processError(response, plotName) {
+        var parsedResponse = JSON.parse(response);
+        if (parsedResponse.hasOwnProperty("type") && parsedResponse['type'] == 'ERROR') {
+            this.setState({
+                dialogTitle: "NetPyNE returned an error plotting " + plotName,
+                dialogMessage: parsedResponse['message'] + "\n " + parsedResponse['details'],
+                openDialog: true
+            });
+            return true;
+        }
+        return false;
+    }
+
+    plotFigure(plotName, plotMethod, flavour=false) {
```

Comment:
maybe call it `plotType` or similar instead of `flavour` (with the British spelling!)

Reply:
I'm learning new spellings just in case something happens in the future.